Dockerfile

```diff
@@ -7,5 +7,6 @@ RUN mvn -f /home/app/pom.xml clean test package
 # Package stage
```

Comment:
Could we also add documentation to the README about where the swagger endpoint lives? Also could we update the README `Rapid Testing Instructions` to include setting the new `system.properties` file?